app/controllers/games_controller.rb

```diff
@@ -37,14 +40,25 @@ def update
   end
 
   def destroy
-    Game.destroy(params[:id])
-    redirect_to "/games"
+    if Game.find(params[:id])
+      @game = Game.find(params[:id])
+      if current_user.id == @game.user_id
```

Comment:
You can check for games owned by a user during the `find` call:

```ruby
game = Game.find_by(user: current_user, id: params[:id])
```

Another alternative is to take advantage of the `has_many :games` association on the `User` model:

```ruby
game = current_user.games.find(params[:id])
```

This will only search within the list of games that the user has created.